packages/twenty-front/src/modules/serverless-functions/utils/getFunctionInputFromSourceCode.ts

```diff
@@ -0,0 +1,27 @@
+import { getFunctionInputSchema } from '@/serverless-functions/utils/getFunctionInputSchema';
+import { getDefaultFunctionInputFromInputSchema } from '@/serverless-functions/utils/getDefaultFunctionInputFromInputSchema';
+import { FunctionInput } from '@/workflow/types/FunctionInput';
+import { isObject } from '@sniptt/guards';
+import { isDefined } from 'twenty-ui';
+
+export const getFunctionInputFromSourceCode = (
+  sourceCode?: string,
+): FunctionInput => {
+  if (!isDefined(sourceCode)) {
+    throw Error('Source code is not defined');
```

Comment:
Calling `Error()` returns a new `Error` instance, but it would be more idiomatic to call `throw new Error()`. For some people, this is not a better pattern. However, it is most commonly used across the codebase.

Reply:
yes that's a typo thanks!

Reply:
@martmull can we throw a specific error for Serverless there?